### PR TITLE
Handle testable refactor

### LINKED ISSUE #1799: Handle Testable Refactor for TestResult
### Similar requests

- [X] I have searched and found no existing similar requests

### How are you using Legend?

_No response_

### What problems are you trying to solve?

Hanlde Testable Refactor on TestResult to have only TestExecuted and TestError 
https://github.com/finos/legend-engine/pull/1238/files

We will need to adjust on how we read test results to display in the service test editor and the Global Test Runner

### Describe the solution you would like to see implemented

Remove TestPass/TestFailed add TestExecuted 
https://user-images.githubusercontent.com/19693874/208887044-fe22d26b-09ad-4a74-abcc-18c622880914.png

### Describe alternatives you have considered

_No response_

### Documentation, Design, Adoption, Migration Strategy

_No response_

### Contribution

- [X] I would like to work on this feature

### PR DESCRIPTION
**Pull request by gayathrir11**
## Summary

closes #1799 
depends on finos/legend-engine#1238

## How did you test this change?

- [ ] Test(s) added
- [x] Manual testing (please provide screenshots/recordings)
- [ ] No testing (please provide an explanation)

### CODE REVIEW

**Review by YannanGao-gs — APPROVED** [review]

File: packages/legend-graph/src/graphManager/protocol/pure/v1/V1_PureGraphManager.ts [hunk 1]
```diff
@@ -1786,7 +1787,7 @@ export class V1_PureGraphManager extends AbstractPureGraphManager {
       const runTestsInput = new V1_RunTestsInput();
       runTestsInput.model = this.getFullGraphModelData(graph);
       const runTestableInput = new V1_RunTestsTestableInput();
-      const unitAtomicTest = new V1_AtomicTestId();
+      const unitAtomicTest = new V1_UniqueTestId();
```
Comment (MauricioUyaguari) [inline packages/legend-graph/src/graphManager/protocol/pure/v1/V1_PureGraphManager.ts:1790]: did we double check that generate expected result and generate test data is working as expected with this change?

File: .changeset/moody-tips-give.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+'@finos/legend-extension-dsl-persistence': patch
+'@finos/legend-application-studio': patch
+'@finos/legend-graph': patch
+---
+
+Handle testable refactor
```
Comment (MauricioUyaguari) [inline .changeset/moody-tips-give.md:7]: I think we can give a better description here. Let's give maybe one or two sentences on the main model changes we are doing on Testables. Thanks!

**Review by MauricioUyaguari — CHANGES_REQUESTED** [review]
Small Changes.

File: packages/legend-graph/src/graphManager/protocol/pure/v1/V1_PureGraphManager.ts (hunk 1 above)
Comment (gayathrir11) [reply packages/legend-graph/src/graphManager/protocol/pure/v1/V1_PureGraphManager.ts:1790]: tested generate expected result. But couldn't test generate test data as the api is not open sourced

**Review by MauricioUyaguari — APPROVED** [review]

**Review by MauricioUyaguari — APPROVED** [review]

**Pull request merged**